oaknational/Oak-Web-Application
# [CUR-265] Switch priority order of tabs for curriculum pages

## Description

- Updated initial link to go to units page instead of overview page
- Switched order of tabs in the curriculum header

## How to test

1. Go to https://deploy-preview-1989--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/overview
2. Click on any subject/phase, where it should lead you to the sequence page first
3. You should see the order of tabs have also been switched

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-10-09 at 11 49 39](https://github.com/oaknational/Oak-Web-Application/assets/32605982/119293e2-74ae-4c6a-b9e5-4f743fb5cd85)


How it should now look:
<img width="1003" alt="Screenshot 2023-10-09 at 11 49 00" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/01523a8b-a15f-4e73-a98a-8271e78beca6">

## Checklist
